Feature/srl 392  text paragraph

sleepingmonk opened the pull request.

sleepingmonk committed:
Merge pull request #329 from chapter-three/feature/SRL-348--ga-gtm-form-tracking

Feature/srl 348  ga gtm form tracking

sleepingmonk committed:
SRL-392: Add text paragrph.

robdecker commented on web/themes/custom/atrium/templates/paragraph/paragraph--text.html.twig:
```diff
@@ -0,0 +1,49 @@
+  <div class="jcc-text-section__heading margin-bottom-4">
+    <h2 class="h-dash text-primary-dark">{{ content.field_heading }}</h2>
+  </div>
+  <div class="jcc-text-section__body usa-prose">
```
What do you think about using a `organisms/sections/text-section`?

robdecker replied:
http://dev.patternlab.courts.ca.gov/?p=organisms-text-section

sleepingmonk committed:
SRL-392: switch to text-section component

I didn't realize when I first tried it that the column of text would be
restricted in width by usa-prose.  It wasn't because of the template.
